core/scripts/functions/src/sample_keys.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "EthAddress": "0x93157d6Cb4b1CA26E733072D8021947F05ed4942",
```

Comment:
Not sure if it makes sense to put any values in here. Somebody might think that they could work for their new DON while in reality they need to fetch all of it from their nodes.